filter-commits.js

```diff
@@ -0,0 +1,23 @@
+// filter-commits.js
+const ignorableCommits = [
```

Comment:
This array should be empty by default but should be overwritable by a environmant variable like all the other settings.